src/App.js

```diff
@@ -1,7 +1,34 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import './App.css';
+import { useDispatch } from 'react-redux';
+import AdventureList from './Components/AdventureList';
+
+import { authenticateUser } from './redux/slice/userSlice';
+import checkCurrentUser from './redux/actions/userActions';
+
+export const currentUserObj = () => {
+  if (localStorage.getItem('id') !== null) {
+    const userObj = localStorage.getItem('id');
+
+    return JSON.parse(userObj);
+  }
+  return false;
+};
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed (I found 7):  

![image](https://github.com/bilalrajput09/Outdoor-Adventure-Bookings/assets/57421823/d3bcaaa5-84be-4744-b116-20ade5bbb9ff)


Shouldn't take you more than 5-10 min so don't worry.  

That way, your code will meet the latest standards. 💪